### PR TITLE
New_MinimOSD_firmware_branch

### PR DESCRIPTION
**Pull request by lvale**
Complete reference to MicroOSD and Night_Shift

### CODE REVIEW

File: common/source/docs/common-minim-osd-quick-installation-guide.rst [hunk 1]
```diff
@@ -1,63 +1,128 @@
...
+
+
+
+
+
+
+
+
+
+    Firmware available for MinimOSD
```
Comment (hamishwillee) [inline common/source/docs/common-minim-osd-quick-installation-guide.rst]: Nothing below here will build. This is obvious if you look at the preview of this page.

Recommend you build and test this as per http://ardupilot.org/copter/docs/common-wiki_editing_guide.html#making-a-big-edit

The specific problem is indentation. In reStructuredText indentation means something. Everything should be left-aligned unless you mean it not to be.

**Comment by hamishwillee** [comment]
Hi Luis

No need to close. Id much rather you got used to the toolchain, but if you can't then I'll integrate it! Or was this closed for another reason?

There is also more information on installing Sphinx natively on Mac here: http://freelancing-gods.com/thinking-sphinx/installing_sphinx/mac.html

**Comment by lvale** [comment]
I'll try to get into Sphinx, but the changes I suggest I think add value to the community, so if you don't mind please publish.

**Comment by hamishwillee** [comment]
Great! Can you please reopen this? If not I'll have to duplicate your stuff manually.

**Comment by lvale** [comment]
Done here https://github.com/ArduPilot/ardupilot_wiki/pull/359

Thanks

**Pull request closed without merging**